src/Avalonia.Base/StyledElement.cs

```diff
@@ -816,8 +819,11 @@ private void ApplyStyles(IStyleHost host)
             
             if (host.IsStylesInitialized)
             {
-                foreach (var style in host.Styles)
-                    ApplyStyle(style, host, FrameType.Style);
+                var styles = host.Styles;
+                for (var i = 0; i < styles.Count; ++i)
```

Comment:
i++? instead of ++i,

maybe it doesnt make a difference?

Reply:
A while ago "++i" was usually faster. I am not sure if it matters today.

Reply:
They produce the same MSIL for the loop.